lib/dotcom_web/live/trip_planner.ex

```diff
@@ -88,7 +90,65 @@ defmodule DotcomWeb.Live.TripPlanner do
     """
   end
 
+  defp itinerary_panel(%{details_index: details_index} = assigns) do
```

Comment:
I think I want to pull `itinerary_panel` out into a separate live component, and have _it_ handle the summary versus details toggling, rather than having that logic handled in the top-level `TripPlanner` live.

The one complicating factor is the map, since on mobile the map gets rendered in the middle of the details panel. That complicating factor is why I left it like this in this PR, but I sort of do want to figure out at least some strategy for this before merging.

Reply:
I wound up pulling out `TripPlannerResultsSection` instead, which includes the map. I think this is a passable solution for now, and might wind up being actually good? Interested to know what you think.